lms/djangoapps/grades/new/course_grade.py

```diff
@@ -51,6 +52,9 @@ def locations_to_weighted_scores(self):
         for chapter in self.chapter_grades:
             for subsection_grade in chapter['sections']:
                 locations_to_weighted_scores.update(subsection_grade.locations_to_weighted_scores)
+        log.warning("calculated locations_to_weighted_scores property: {0}".format(
+            unicode(locations_to_weighted_scores)
+        ))
```

Comment:
This could be a rather large dictionary.  Do we want to log that much information?  Especially at warning level.  Maybe just "calculated locations_to_weighted_scores property for {course_key}"